### PR TITLE
Feature/contentchanges

### PR DESCRIPTION
**Pull request by govTechSatish**

### CODE REVIEW

File: app/views/vet/practice.njk [hunk 1]
```diff
@@ -17,15 +17,12 @@
     <div class="govuk-grid-column-two-thirds">
       <form method="POST">
         <input type="hidden" name="crumb" value="{{crumb}}"/>
-        <h1 class="govuk-heading-l">What is the name of the practice the vet works for?</h1>
+        <h1 class="govuk-heading-l">What is the name of the vet practice the vet working for at the time of the review?</h1>
```
Comment (st3v3nhunt) [inline app/views/vet/practice.njk:20]: missing `was`...
`the vet working` => `the vet was working`

File: app/views/vet/email.njk [hunk 2]
```diff
@@ -33,12 +33,14 @@
           errorMessage: errorMessage
         }) }}
 
+        <p>We'll send you a link to this email for the vet to record information about the review.</p>
```
Comment (st3v3nhunt) [inline app/views/vet/email.njk:36]: This isn't needed.

File: app/views/vet/email.njk [hunk 3]
```diff
@@ -17,14 +17,14 @@
     <div class="govuk-grid-column-two-thirds">
       <form method="POST">
         <input type="hidden" name="crumb" value="{{crumb}}"/>
-        <h1 class="govuk-heading-l">What is the vet practice email address?</h1>
 
+        <h1 class="govuk-heading-l">What is the vet practice email address?</h1>
         {{ govukInput({
           label: {
-            text: "Vet practice email address"
+            text: "What is the vet practice email address?"
```
Comment (st3v3nhunt) [inline app/views/vet/email.njk:24]: This doesn't need changing.

File: app/views/vet/confirmation.njk [hunk 4]
```diff
@@ -11,13 +11,13 @@
         html: "Booking reference number<br><strong>" + reference + "</strong>"
       }) }}
 
-      <p>The record has been submitted.</p>
       <h2 class="govuk-heading-m">What happens next</h2>
-      <p class="govuk-body">We’ve contacted the farmer to ask them to check and confirm the record. They can claim their funding when they have confirmed this.</p>
+      <p class="govuk-body">We’ve contacted the farmer to tell them they can claim their funding.</p>
       <p class="govuk-body">Invoice the farmer if you have not already done this.</p>
       <h2 class="govuk-heading-m">Contact the Rural Payments Agency</h2>
       <p class="govuk-body">If you have a question about your submission, call the Rural Payments Agency on 03000 200 301</p>
```
Comment (st3v3nhunt) [inline app/views/vet/confirmation.njk:18]: A full stop needs adding to the end of the sentence.

File: app/views/vet/confirmation.njk [hunk 5]
```diff
@@ -11,13 +11,13 @@
...
       <p class="govuk-body">Monday to Friday 8.30am to 5pm, except bank holidays <a href="https://www.gov.uk/call-charges" target="_blank">Find out about call charges</a>.</p>
```
Comment (st3v3nhunt) [inline app/views/vet/confirmation.njk:19]: Find out about call charges in a new paragraph.

File: app/views/vet/confirmation.njk [hunk 6]
```diff
@@ -11,13 +11,13 @@
...
+      <p class="govuk-body"><a href="https://www.gov.uk/call-charges" target="_blank">What did you think of this service?</a>(takes 30 seconds)</p>
```
Comment (st3v3nhunt) [inline app/views/vet/confirmation.njk:20]: I don't think the href should be `../call-charges`, please update to whatever it should be. There needs to be a space before the opening paren.

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
A few changes please.

I think the `further investigation required` option needs to be removed from the dairy-test route, in the same way as the beef-test has been altered. And that will probably mean the `yes-no-investigate-radios` module can be removed too.

The `check-answers` route view has the answers in sentence case. Currently they are lower cased.

File: app/views/vet/confirmation.njk (hunk 6 above)
Comment (govTechSatish) [reply app/views/vet/confirmation.njk:20]: Thats not been given yet. I have asked Joy this question in ticket as well.

File: app/views/vet/confirmation.njk (hunk 5 above)
Comment (govTechSatish) [reply app/views/vet/confirmation.njk]: done

File: app/views/vet/email.njk (hunk 3 above)
Comment (govTechSatish) [reply app/views/vet/email.njk]: done

File: app/views/vet/confirmation.njk [hunk 7]
```diff
@@ -16,8 +16,9 @@
       <p class="govuk-body">Invoice the farmer if you have not already done this.</p>
       <h2 class="govuk-heading-m">Contact the Rural Payments Agency</h2>
       <p class="govuk-body">If you have a question about your submission, call the Rural Payments Agency on 03000 200 301</p>
-      <p class="govuk-body">Monday to Friday 8.30am to 5pm, except bank holidays <a href="https://www.gov.uk/call-charges" target="_blank">Find out about call charges</a>.</p>
-      <p class="govuk-body"><a href="https://www.gov.uk/call-charges" target="_blank">What did you think of this service?</a>(takes 30 seconds)</p>
+      <p class="govuk-body">Monday to Friday 8.30am to 5pm, except bank holidays</p>
```
Comment (st3v3nhunt) [inline app/views/vet/confirmation.njk:19]: A full stop needs adding to the end of the sentence.

File: app/views/vet/confirmation.njk (hunk 4 above)
Comment (st3v3nhunt) [reply app/views/vet/confirmation.njk:18]: I can't see any change made to this.

File: app/views/vet/confirmation.njk [hunk 8]
```diff
@@ -16,8 +16,9 @@
...
+      <p> <a href="https://www.gov.uk/call-charges" target="_blank">Find out about call charges</a>.</p>
```
Comment (st3v3nhunt) [inline app/views/vet/confirmation.njk:20]: Remove the empty space at the start of the `p` tag.
No full stop required.

File: app/views/vet/confirmation.njk [hunk 9]
```diff
@@ -16,8 +16,9 @@
...
+      <p class="govuk-body"><a href="#" target="_blank">What did you think of this service?</a>(takes 30 seconds)</p>
```
Comment (st3v3nhunt) [inline app/views/vet/confirmation.njk:21]: Space need added before the paren.

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]

File: app/views/vet/confirmation.njk (hunk 4 above)
Comment (govTechSatish) [reply app/views/vet/confirmation.njk:18]: its already there.

File: app/views/vet/confirmation.njk (hunk 7 above)
Comment (govTechSatish) [reply app/views/vet/confirmation.njk]: done

File: app/views/vet/confirmation.njk (hunk 9 above)
Comment (govTechSatish) [reply app/views/vet/confirmation.njk]: done

**Review by st3v3nhunt — APPROVED** [review]
LGTM

**Pull request merged**